src/devtools/store.js

```diff
@@ -782,10 +818,13 @@ export default class Store extends EventEmitter {
             });
 
             element.weight = nextWeight;
-
-            weightDelta = nextWeight - prevWeight;
+            // TODO: passing null here is suspicious, but it's existing behavior.
+            // It is suspicious because either we shouldn't need a delta at all--
+            // or we should apply it to all parents, and not just this item.
+            this._adjustParentTreeWeight(null, nextWeight - prevWeight);
```

Comment:
We should no longer need to call this during a reorder, right? Especially now that we have invariants in place to ensure the contents didn't change, only their order.

Just to be sure, I commented out this whole `if (!element.isCollapsed)` block in both master _and_ your branch, and all of the tests  pass.

Reply:
Sure